code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -415,7 +415,23 @@ GLOBAL_VAR(bomb_set)
 	if(timing)	//boom
 		INVOKE_ASYNC(src, .proc/explode)
 		return
-	qdel(src)
+
+    //if no boom then we need to let the blob capture our nuke
+	var/turf/T = get_turf(src)
+	if(!T)
+		return
+	if(locate(/obj/structure/blob) in T)
+		return
+	var/obj/structure/blob/normal/N = new /obj/structure/blob/normal(loc, min(obj_integrity, 30))
```

Comment:
I don't see any reason to first make this a normal tile and then change it to a `captured_nuke` tile.
Neither `/obj/structure/blob` nor `/obj/structure/blob/normal` take any additional parameters as well. So the second param here should not do anything as far as I can tell.